pset-iterators.js

```diff
@@ -183,7 +208,46 @@ Expected Result Shape: An array of objects (with each object representing a pres
 console.log('Problem 7:')
 
 // Add your code below this line
+// console.log(presidents[44])
+
+// creating function to address null value for trump's left_office date
+// really brittle -- works until he leaves office
+// but sufficient for the array we've been given
+// and i find it more interesting than hardcoding
+function leftOfficeDateChanger(array) {
+  for (i = 0; i < array.length; i++) {
+    if (array[i].left_office === null) {
+      array[i].left_office = new Date().toISOString().slice(0,10)
+    }
```

Comment:
This certainly works! Another potential solution is to use an iterator method like filter (to remove edge cases like a president still in office). [Here](https://github.com/jsd20200127/hw-03-iterators-solution/blob/c31d6d65d8f18a945cb6e59470aca10687d61aa3/pset-iterators.js#L204-L206) is an example of that.